src/converter/one/jfr/JfrReader.java

```diff
@@ -743,4 +739,9 @@ private boolean ensureBytes(int needed) throws IOException {
         buf.flip();
         return buf.limit() > 0;
     }
+
+    public long eventTimeToNanos(long time) {
+        long nanosSinceChunkStart = (long) ((time - chunkStartTicks) * nanosecondsPerTick);
+        return chunkStartNanos + nanosSinceChunkStart;
```

Comment:
I see you like excessively long variable names and extra temporary variables )
What about
```
return chunkStartNanos + (long) ((time - chunkStartTicks) * nanosPerTick);
```